src/Augurk/Scripts/Application/filters.js

```diff
@@ -97,15 +96,33 @@ nomnoml.draw(canvas, source);</script></div>`;
 
                 return result;
             }
-        ));
+        );
+
+        output = output.replace(/\<pre><code class=\"sequence language-sequence\"\>(?:Label\:\W?(.*$))?([\s\S]*?)<\/code><\/pre>/gm,
+            function (entireString, title, match) {
+                var id = 'seq' + uniqueIntegerService.getNextInteger().toString();
+                var result = `<div class="sequence" id="` + id + `">
+<script>
+var source = \`` + _.unescape(match) + `\`;
+var diagram = Diagram.parse(source);
+diagram.drawSVG("` + id + `", {theme: 'snapSimple'});</script></div>`;
```

Comment:
Is it an explicit choice to only support snapSimple?
We could make this configurable...

Reply:
We won't do this right now. Let's wait and see if there's any demand for it first.